ipamd/datastore/data_store.go

```diff
@@ -403,7 +403,6 @@ func (ds *DataStore) RemoveUnusedENIFromStore(warmIPTarget int) string {
 
 	deletableENI := ds.getDeletableENI(warmIPTarget)
 	if deletableENI == nil {
-		log.Debugf("No ENI can be deleted at this time")
```

Comment:
Yes, it's usually not true, and if happened to be, it would be a duplicate. We log the real failure reason for the error before we return in [`ds.getDeletableENI()`](https://github.com/aws/amazon-vpc-cni-k8s/blob/a8a7c49641399dfded4f9fedf8be588b293f6dea/ipamd/datastore/data_store.go#L334-L362)